python/src/apx_agent/cli.py

```
@@ -5837,6 +5837,131 @@ def _kill_pid(pid: int, name: str) -> bool:
+    # --- resolve question ---
+    if not question:
+        if not click.get_text_stream("stdin").isatty():
+            question = click.get_text_stream("stdin").read().strip()
+        if not question:
+            raise click.ClickException("Provide a question as an argument or via stdin.")
+
+    # --- auth ---
+    headers: dict[str, str] = {"Content-Type": "application/json"}
+    if profile:
+        try:
+            result = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)